### PR TITLE
[MLU] add huber_loss kernel.

### PR DESCRIPTION
**Pull request by ShawnNew**
### PR types

New features

### PR changes

OPs

### Describe

add huber loss for mlu, depends on [PR46426](https://github.com/PaddlePaddle/Paddle/pull/46429)

### CODE REVIEW

File: paddle/fluid/operators/huber_loss_op_mlu.cc [hunk 1]
```diff
@@ -0,0 +1,187 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (ronny1996) [inline paddle/fluid/operators/huber_loss_op_mlu.cc]: 2022

**Review by ronny1996 — CHANGES_REQUESTED** [review]

File: paddle/fluid/operators/huber_loss_op_mlu.cc (hunk 1 above)
Comment (ShawnNew) [reply paddle/fluid/operators/huber_loss_op_mlu.cc]: Done

**Review by qili93 — DISMISSED** [review]
LGTM

File: paddle/fluid/operators/huber_loss_op_mlu.cc [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/fluid/framework/op_registry.h"
+#include "paddle/fluid/operators/mlu/mlu_baseop.h"
+
+namespace paddle {
+namespace operators {
+
+using Tensor = framework::Tensor;
```
Comment (chenwhql) [inline paddle/fluid/operators/huber_loss_op_mlu.cc]: 麻烦直接using Tensor = phi::DenseTensor; 或者只用使用 phi::DenseTensor，这里有历史遗留问题，正在全局删除

https://github.com/PaddlePaddle/Paddle/pull/46432

Comment (ShawnNew) [reply paddle/fluid/operators/huber_loss_op_mlu.cc]: Done

**Review by qili93 — APPROVED** [review]
LGTM

**Review by ronny1996 — APPROVED** [review]

**Review by chenwhql — APPROVED** [review]

**Pull request merged**